core/history/historymode.go

```diff
@@ -27,22 +27,22 @@ import (
 type HistoryMode uint32
 
 const (
-	// AllHistory (default) means that all chain history down to genesis block will be kept.
-	AllHistory HistoryMode = iota
+	// KeepAll (default) means that all chain history down to genesis block will be kept.
+	KeepAll HistoryMode = iota
 
-	// PostMergeHistory sets the history pruning point to the merge activation block.
-	PostMergeHistory
+	// KeepPostMerge sets the history pruning point to the merge activation block.
+	KeepPostMerge
 )
 
 func (m HistoryMode) IsValid() bool {
```

Comment:
nitpick, i would prefer to rename it to `Mode`. Combined with the package name it's `history.Mode`